test/studies/shootout/submitted/revcomp2.good

```diff
@@ -1,6 +1,3 @@
-revcomp2.chpl:19: warning: The 'iohints' type is deprecated; please use a variant of 'file.writer' that takes an 'ioHintSet' instead.
```

Comment:
This tests `.perfkeys` line number also needs to be updated to reflect the missing deprecation message. (not important now because of the notest, but I thought I'd point it out anyway).

Reply:
Ooooh, good catch.  Thanks!